File: chapter12-AdvancedPython1/expression.cpp
#include<iostream>
using namespace std;

int main() {
    int a;
    cout << "Enter first number:";
    cin >> a;
    int b;
    cout << "Enter second number:";
    cin >> b;
    int result = a / b;
    if(a % b != 0){
        cout << "the result is " << result << " and the reminder is " << a % b << endl;
    }
    else{
        cout << "the result is " << result << endl;
    }
}

File: chapter12-AdvancedPython1/expression.py
a = int(input("Enter first number:"))
b = int(input("Enter second number:"))
result = int(a / b)
if(a % b != 0):
    print(f"The result is {result} and reminder is {a % b}")
else:
    print(f"Thw result is {result}")